src/Compilation.zig

```diff
@@ -5908,8 +5908,7 @@ fn updateWin32Resource(comp: *Compilation, win32_resource: *Win32Resource, win32
         // them being defined matches the behavior of how MSVC calls rc.exe which is the more
         // relevant behavior in this case.
         switch (rc_src.owner.optimize_mode) {
-            .Debug => try argv.append("-D_DEBUG"),
-            .ReleaseSafe => {},
+            .Debug, .ReleaseSafe => {},
```

Comment:
@squeek502 I'm unsure if I should leave this here?

Reply:
In double checking, I'm not actually sure how I originally determined that MSVC sets this when calling `rc.exe`. I think that might actually be a CMake thing, not a MSVC thing. The default "Windows Desktop Application" template in Visual Studio uses `/MDd` and `_DEBUG` but does not set it for `rc.exe`.

So, removing the `-D_DEBUG` here is just the correct thing to do generally regardless of the other changes in the PR.